lib/mods/replace-constructors.js

```diff
@@ -3,6 +3,30 @@
 const nodeTypes = require('ts-morph').TypeGuards;
 const globalsSet = require('../globals-set.js');
 
+/**
+ * Adds an import for the module to the list
+ *
+ * @param {string} mod - the name of the module to import from
+ * @param {string} original - the name of the original to get
+ * @param {Object} imports - the current list of imports
+ */
+function updateImports(mod, original, imports) {
+  if (mod in imports) {
+    if (original in imports[mod]) {
```

Comment:
Is this code tested? It seems like it could be via a test file that does `new Something()` twice.